lightning/src/chain/chaininterface.rs

```diff
@@ -176,7 +176,7 @@ pub trait FeeEstimator {
 }
 
 /// Minimum relay fee as required by bitcoin network mempool policy.
-pub const MIN_RELAY_FEE_SAT_PER_1000_WEIGHT: u64 = 4000;
+pub const MIN_RELAY_FEE_SAT_PER_1000_WEIGHT: u64 = 253;
```

Comment:
Could we delete this in favor of `FEERATE_FLOOR_SATS_PER_KW` below ? Different types, but same number.

Reply:
Thank you where did you find them ? I've been grepping the bitcoin repo, can't see them.

I agree these are two different settings - I'd suggest we clarify one of them is the incremental fee rate, the other is the broadcast fee rate - "MIN_RELAY_FEE" and "FEERATE_FLOOR" is a little too similar.

Reply:
could use the names from bitcoin core, `minrelaytxfee` and `incrementalrelayfee`